src/main/java/com/maxrt/shoeshop/shoes/Shoes.java

```diff
@@ -0,0 +1,13 @@
+package com.maxrt.shoeshop.shoes;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor(access = AccessLevel.PUBLIC)
+public class Shoes {
+    protected ShoeType type;
```

Comment:
why these fields are protected?

Reply:
Because those fields were protected in my UML diagram